backend/src/core/middlewares/file-attachment.middleware.ts

```diff
@@ -11,6 +13,9 @@ const fileUploadHandler = fileUpload({
   },
   abortOnLimit: true,
   limitHandler: function (_: Request, res: Response) {
+    // not sure whether to amend to use dependency injection so as to
+    // 1. use logger to log these errors (currently these are unlogged?)
```

Comment:
All request and responses are logged rn so it shouldn't be an issue. The only potential problem is that we're logging the whole request including the file field - rendering the log record too big and thus truncated => not valid JSON for DataDog to index => We might wanna look into excluding these file fields from the logs to prevent this

Reply:
ohh where is this logging being done? is this an issue to be tackled in a separate PR or part of this PR?

Reply:
Separate PR ser

Reply:
OK created [issue](https://github.com/opengovsg/postmangovsg/issues/1788) (and added to [Notion](https://www.notion.so/opengov/Exclude-unnecessary-fields-from-request-to-enable-Datadog-indexing-e1d44a0dd74c4a5388a80f8616765677))

I love creating chore tasks loool